var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -130,17 +133,15 @@ class Sundials(CMakePackage):
     # External libraries incompatible with single precision
     conflicts('+klu',   when='precision=single')
     conflicts('+hypre', when='+hypre@:2.12.0 precision=single')
+    conflicts('+superlu-dist', when='precision=single')
 
     # External libraries incompatible with extended (quad) precision
     conflicts('+lapack',     when='precision=extended')
     conflicts('+superlu-mt', when='precision=extended')
+    conflicts('+superlu-dist', when='precision=extended')
```

Comment:
Add a conflict for `+superlu-dist` with versions before 5.

Reply:
Took care of this in the depends_on now. We actually require 6.1.1 or higher.

Reply:
Sorry I meant that `+superlu-dist` should conflict with sundials versions earlier than 5. I'd missed the SuperLU_DIST version requirement, good catch on that one.